<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="error_msg" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-60</y>
     <width>751</width>
     <height>601</height>
    </rect>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLineEdit" name="admin_info">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>270</y>
      <width>251</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="back_btn">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>500</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:qlineargradient(spread:pad , x1:0, y1:0, x2:1, y2:0, stop:0 rgba(255,178,102,0), stop:o.55 rgba(235,148,61,0), stop:0.98 rgba(0,0,0,0),stop:1 rgba(0,0,0,0));

border-style:outset;

border-width:2px;

border-radius:10px;

border-color:white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/back_icon.jpeg</normaloff>images/back_icon.jpeg</iconset>
    </property>
   </widget>
   <widget class="QComboBox" name="del_opt">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>230</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox:item:selected{
background-color: rgb(153, 193, 241);}</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>Choose Delete Option</string>
    </property>
   </widget>
   <widget class="QPushButton" name="delete_btn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>360</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);
border-style:outset;
border-width:2px;
border_radius:10px;
background-color:rgb(165,4,4);
color:white;
font: bold 14px;
padding:6px;
min-width:10px;
</string>
    </property>
    <property name="text">
     <string>DELETE</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>320</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 0, 0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 63 11pt &quot;URW Bookman&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:16pt; font-weight:600; color:#5555ff;&quot;&gt;Delete Admin&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="home_btn">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/home_icon.jpeg</normaloff>images/home_icon.jpeg</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>51</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>niit.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>266</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font:12pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Enter Admin nformation</string>
    </property>
   </widget>
   <widget class="QPushButton" name="logout_btn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>90</y>
      <width>21</width>
      <height>21</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">border-style:outset;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>images/logout_icon.png</normaloff>images/logout_icon.png</iconset>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
